### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by ErdalNayir**
I love this game. I hope that translation which I have done make this game better.

### CODE REVIEW

File: values-tr/strings.xml [hunk 1]
```diff
@@ -2493,106 +2494,106 @@ Press a key to assign it to this action.</string>
...
-    <string name="stage_files_actionrename">Rename</string>
-    <string name="stage_files_actionrename_text">Rename file %1$s:</string>
-    <string name="stage_files_renamecollision_title">Naming collision</string>
-    <string name="stage_files_renamecollision_text">The file %1$s does already exist!</string>
-    <string name="stage_files_actiondelete">Delete</string>
-    <string name="stage_files_actiondelete_title">Delete?</string>
+    <string name="stage_files_actionrename">Yeniden adlandır</string>
+    <string name="stage_files_actionrename_text">Dosyayı yeniden adlandır %1$s:</string>
+    <string name="stage_files_renamecollision_title">İsim çakışması</string>
+    <string name="stage_files_renamecollision_text"> %1$s isimli dosya zaten var!</string>
```
Comment (JustAnyones) [inline values-tr/strings.xml:2513]: I'm not sure you've intended this whitespace

File: values-tr/strings.xml [hunk 2]
```diff
@@ -2493,106 +2494,106 @@ Press a key to assign it to this action.</string>
...
-    <string name="stage_files_createfolder_cmdcreate">Create</string>
-    <string name="stage_files_createfoldercollision_title">File exists</string>
-    <string name="stage_files_createfoldercollision_text">The file %1$s does already exist!</string>
+    <string name="pluginerror_locateplugin">Plugin\'i konumlandır</string>
+    <string name="stage_files_createfolder">Dosya oluştur</string>
+    <string name="stage_files_createfolder_title">Yeni dosya</string>
+    <string name="stage_files_createfolder_text">Dosya için isim girin:</string>
+    <string name="stage_files_createfolder_cmdcreate">Oluştur</string>
+    <string name="stage_files_createfoldercollision_title">Dosya var</string>
+    <string name="stage_files_createfoldercollision_text"> %1$s isimli dosya zaten var</string>
```
Comment (JustAnyones) [inline values-tr/strings.xml:2532]: Same here

File: values-tr/strings.xml (hunk 1 above)
Comment (ErdalNayir) [reply values-tr/strings.xml:2513]: Sorry it's not my intention to leave whitespace

File: values-tr/strings.xml (hunk 2 above)
Comment (ErdalNayir) [reply values-tr/strings.xml:2532]: I can fix this issue

File: values-tr/strings.xml (hunk 1 above)
Comment (ErdalNayir) [reply values-tr/strings.xml:2513]: I just want to clarify something. You are talking about whitespace at the beginning of sentences right? from 2513 and 2532

Comment (JustAnyones) [reply values-tr/strings.xml]: Correct.

**Comment by JustAnyones** [comment]
The rest looks good, thanks for your contribution :)

**Comment by ErdalNayir** [comment]
You are welcome 👍🏻

**Pull request merged**